lib/transform/pathRegexTransformer.ts

```diff
@@ -62,7 +62,7 @@ const buildPathRegex = (
   params.forEach((v, i) => {
     if (path.startsWith(`/{${v}}`) && pathParams.get(v)) {
       // We allow first param in path as multi path param
-      path = path.replace("{" + v + "}", "(.*)");
+      path = path.replace("/{" + v + "}", "(.*)");
```

Comment:
How to ensure it matches to the path of `/providers/xxx` firstly if it's defined for tenant scope?